packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -90,6 +90,8 @@ const isTestMode = !!(
   process.env.DEBUG
 )
 
+const sessionId = Math.round(Math.random() * 100 + Date.now())
```

Comment:
Why Date + random 0-100ms?

```suggestion
const sessionId = Math.floor(Number.MAX_SAFE_INTEGER * Math.random())
```

Reply:
It's what we do for client ids: https://github.com/vercel/next.js/blob/8f2dd2096789336a935e335fec095ac2dcde775b/packages/next/src/client/components/react-dev-overlay/app/hot-reloader-client.tsx#L48